Content.Shared/Body/Systems/SharedBodySystem.Organs.cs

```diff
@@ -183,6 +183,32 @@ public bool RemoveOrgan(EntityUid organId, OrganComponent? organ = null)
         return list;
     }
 
+    /// <summary>
+    /// Returns a list of Entity<<see cref="T"/>, <see cref="OrganComponent"/>>
+    /// for each organ of the body
+    /// </summary>
+    /// <typeparam name="T">The component that we want to return</typeparam>
+    /// <param name="bodyUid">The EntityUid of the body to check the organs of</param>
+    /// <param name="body">The BodyComponent of the body to check the organs of</param>
+    public List<Entity<T, OrganComponent>> GetBodyOrganEntityComps<T>(
+        EntityUid bodyUid,
+        BodyComponent? body = null)
```

Comment:
This should take `Entity<BodyComponent?>` instead.

Reply:
alright, sure